packages/eslint-plugin-pf-codemods/src/rules/v6/chartsImportMoved/chartsImport-moved.md

```diff
@@ -0,0 +1,19 @@
+### chartsVictory-promoted [(#11091)](https://github.com/patternfly/patternfly-react/pull/11091)
```

Comment:
We should update the rule name here.

Reply:
Yes, this should match. I renamed the files and didn't notice the comment